### PR TITLE
Added note to README.md that resize is slow in debug mode

### PR DESCRIPTION
**Pull request by soadzoor**
Also added some periods for consistency

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -138,20 +138,20 @@ assert!(subimg.dimensions() == (100, 100));
...
 + **filter3x3**: Perform a 3x3 box filter on the supplied image.
-+ **flip_horizontal**: Flip an image horizontally
-+ **flip_vertical**: Flip an image vertically
-+ **grayscale**: Convert the supplied image to grayscale
++ **flip_horizontal**: Flip an image horizontally.
++ **flip_vertical**: Flip an image vertically.
++ **grayscale**: Convert the supplied image to grayscale.
 + **invert**: Invert each pixel within the supplied image This function operates in place.
-+ **resize**: Resize the supplied image to the specified dimensions
++ **resize**: Resize the supplied image to the specified dimensions. Note that this is very slow in debug mode. Make sure to use release mode when using this.
```
Comment (HeroicKatora) [inline README.md:150]: I believe the slowness is not limited to `resize`, it probably affects most of these methods as they all make use of quite a few layers of abstractions. Maybe append this to the paragraph in line 138?

**Review by HeroicKatora — COMMENTED** [review]
I appreciate the extra effort of fixing the consistency of the list items :smile:

File: README.md (hunk 1 above)
Comment (soadzoor) [reply README.md]: Agreed, pushed another update

**Review by HeroicKatora — APPROVED** [review]

**Comment by HeroicKatora** [comment]
Thank you!

**Pull request merged**